Bridge/cctp/src/handlers/MessageReceived.ts

```diff
@@ -123,4 +146,21 @@ export const MessageReceivedHandler = async (
       id: feeinUSDId,
       feeInUSDC: feeamount,
     });
+
+  const blockTimestamp = Number(block.block_timestamp);
+  const dateFromTimestamp = new Date(blockTimestamp* 1000);
+  const date = dateFromTimestamp.toISOString().split("T")[0];
+  const week = Math.floor(dateFromTimestamp.getTime() / 604800000);
+  const month = dateFromTimestamp.getMonth() + 1; 
+  const year = dateFromTimestamp.getFullYear();
```

Comment:
why calculating it here? Just pass timestamp as a param to all functions.

Reply:
fixed it